front/src/config/i18n/en.json

```diff
@@ -791,6 +791,18 @@
         "saveLabel": "Save configuration"
       }
     },
+    "oauth2": {
+      "buttonConnect": "Connect now",
+      "apiKeyLabel": "Then, enter your client_id and secret_id key here:",
+      "errorEmptyConfig": "Client Id and Secret Id must not be empty !",
```

Comment:
Client secret instead of secret_id.
For more, in english, no space before `!`.

Reply:
I fix it